lib/ModelRoot.js

```diff
@@ -28,11 +28,14 @@ function ModelRoot(o) {
 ModelRoot.prototype.errorSelector = function errorSelector(x, y) {
     return y;
 };
-ModelRoot.prototype.comparator = function comparator(a, b) {
-    if (hasOwn(a, "value") && hasOwn(b, "value")) {
-        return a.value === b.value;
+ModelRoot.prototype.comparator = function comparator(cacheNode, messageNode) {
+    if (hasOwn(cacheNode, "value") && hasOwn(messageNode, "value")) {
```

Comment:
This seems fine for now if we want to merge.

I'm wondering about the $expires check though (the $type check makes complete sense).

If $expires is used to define "equality", then how about other meta-data which may be added to the wrapper? Say, $absolutePath in the new deref world if the only thing which changed for this atom is the absolute path to it, or if the user added their own meta-data to the wrapper, would they need to update the method to account for equality wherever this is used.

Basically just wondering why "expires" is part of "value" equality.